### ISSUE TITLE
Blocked testing with latest version

### ISSUE THREAD

**Issue by frankcapodanno** [searls/jasmine-maven-plugin#348]
With the latest version the plugin is blocked:

[INFO] Executing Jasmine Specs
[INFO] Reading https://chromedriver.storage.googleapis.com/ to seek [chromedriver]
[INFO] Latest version of [chromedriver] is 85.0.4183.87
[INFO] Exporting webdriver.chrome.driver as C:\Users\CapodaFr\.m2\repository\webdriver\chromedriver\win32\85.0.4183.87\chromedriver.exe
Starting ChromeDriver 85.0.4183.87 (cd6713ebf92fa1cacc0f1a598df280093af0c5d7-refs/branch-heads/4183@{#1689}) on port 10942
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
Sep 02, 2020 4:02:37 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMATION: Detected dialect: W3C

**Comment by klieber** [comment]
@frankcapodanno , I just released a new version a few days ago [3.0-beta-01](https://github.com/searls/jasmine-maven-plugin/releases/tag/jasmine-maven-plugin-3.0-beta-01).  Can you see if that works for you?

**Comment by frankcapodanno** [comment]
The problem seems to persist. 
When I use jasmine:bdd goal I receive an error that I don't have permission with Jetty. 

When I use jasmine:test goal the build still remain blocked : 

`Executing Jasmine Specs
ChromeDriver was started successfully.
Sep 20, 2020 3:50:13 PM org.openqa.selenium.remote.ProtocolHandshake createSession
INFORMATION: Detected dialect: W3C`

I tested with : 
OS: Windows 10
Maven : 3.6.2 


That is my  portion of pom.xml that I used: 

           ` <plugin>
                <groupId>com.github.searls</groupId>
                <artifactId>jasmine-maven-plugin</artifactId>
                <version>3.0-beta-01</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>test</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <jsSrcDir>${basedir}/src/main/webapp/js/</jsSrcDir>
                    <jsTestSrcDir>${basedir}/src/test/resources/jstests</jsTestSrcDir>
                    <preloadSources>
                        <source>${basedir}/src/main/webapp/js/simple.js</source>
                    </preloadSources>
                </configuration>
            </plugin>`

[Here](https://github.com/frankcapodanno/simplewarjasmine.git) the complete demo.
![forbidden](https://user-images.githubusercontent.com/8950304/93713198-8f6af000-fb5a-11ea-91b8-cf7651225109.PNG)

**Comment by borncodeguru** [comment]
Hi frankcapodanno, I am also facing same issue. Did you get any solution or workaround for it?

**Comment by frankcapodanno** [comment]
Hi borncodeguru . At the moment I use just the release 2.2 (with jasmine 2.4.1) and just the goal jasmine:bdd . When you have the tests written with the new Jasmine it's not a good workaround !

**Comment by klieber** [comment]
@frankcapodanno not really sure what's happening on your side but I pulled down your demo repository and I don't have any problems.  

I ran `mvn test` and this was the result:

```
[INFO] --- jasmine-maven-plugin:3.0-beta-01:test (default) @ simplewar ---
[INFO] Executing Jasmine Specs
ChromeDriver was started successfully.
Sep 28, 2020 11:16:12 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
[INFO]
-------------------------------------------------------
 J A S M I N E   S P E C S
-------------------------------------------------------
[INFO]
Test Suite
  test spec

Results: 1 specs, 0 failures, 0 pending

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  11.239 s
[INFO] Finished at: 2020-09-28T11:16:14-05:00
[INFO] ------------------------------------------------------------------------
```

I also ran `mvn jasmine:bdd` and it looks fine as well:

![image](https://user-images.githubusercontent.com/868930/94458897-39aacf00-017c-11eb-9676-dc70358a456b.png)

**Comment by frankcapodanno** [comment]
Hi @klieber ,
Do you have tested in Windows 10 with what version of Maven ? 
I will try with an another machine.  When its works I close the Problem.

Many Thanks
Francesco

**Comment by klieber** [comment]
I tested on Windows 10 with the latest version of Maven. I wonder if you have a firewall or something preventing you from opening a port.

**Comment by mdiotude** [comment]
Hi,

At my company we've been having the same issue since the 3.0 alpha release. It only seems to affect the Windows users, on macOS and linux it works fine. In our project we customize the `jsSrcDir`, `jsTestSrcDir`, 'jasmineTargetDir`, and we have a list of `preloadSources`.

We've actually developed a fix for it some time ago. Turns out changing `ResourceHandlerConfiguration:getWelcomeFilePath` from using platform specific file path separators to always forward slash fixed the issue for us.
I tested again with the latest 3.0 beta and I can still reproduce the issue, and the same patch fixes it too.

I'll submit a PR with this change.
